src/main/java/mate/academy/springboot/swagger/controller/ProductController.java

```diff
@@ -0,0 +1,134 @@
+                    content = @Content),
+            @ApiResponse(responseCode = "404", description = "Product not found",
+                    content = @Content) })
+    @GetMapping("/{id}")
+    public ProductResponseDto get(@PathVariable Long id) {
+        return productMapper.mapToDto(productService.getById(id));
+    }
+
+    @Operation(summary = "Delete Product by its id")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "Delete the Product",
+                    content = { @Content(mediaType = "application/json",
+                            schema = @Schema(implementation = Product.class)) }),
+            @ApiResponse(responseCode = "400", description = "Invalid id supplied",
+                    content = @Content),
+            @ApiResponse(responseCode = "404", description = "Product not found",
+                    content = @Content) })
+    @DeleteMapping("/{id}")
+    public void delete(@PathVariable Long id) {
+        productService.delete(id);
+    }
+
+    @Operation(summary = "Enumeration all Products", description = "List of Products from DB")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "successful operation",
+                    content = @Content(array = @ArraySchema(schema =
+                    @Schema(implementation = Product.class)))) })
+    @GetMapping(value = "/all", produces = { "application/json", "application/xml" })
+    public List<ProductResponseDto> findAll(
+            @Parameter(description = "Number of pages. Cannot be empty.",
+                    required = true, schema = @Schema(type = "String", defaultValue = "15"))
+            @RequestParam String numberPage,
+            @Parameter(description = "Count of entries on page. Cannot be empty.",
+                    required = true, schema = @Schema(type = "String", defaultValue = "50"))
+            @RequestParam String countOnPage,
+            @Parameter(description = "Name of sorting column. Cannot be empty.",
+                    required = true, schema = @Schema(type = "String", defaultValue = "id"))
+            @RequestParam String sortBy) {
+        Sort sort = Sort.by(sortUtil.getSort(sortBy));
+        PageRequest pageRequest = PageRequest.of(Integer.parseInt(numberPage.trim()),
+                Integer.parseInt(countOnPage.trim()), sort);
+        return productService.findAll(pageRequest).stream()
+                .map(productMapper::mapToDto)
+                .collect(Collectors.toList());
+    }
+
+    @Operation(summary = "List Products by price range", description = "Returns filtered by price ",
+            tags = { "contact" })
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "successful operation",
+                    content = @Content(schema = @Schema(implementation = Product.class))),
+            @ApiResponse(responseCode = "404", description = "Contact not found") })
```

Comment:
What is contact and how can user get this response with your service implementation?

Reply:
refactored